### PR TITLE
feat(runtime): debug log host function call except gas

### PR DESCRIPTION
**Pull request by ailisp**
This fix the problem in #5062 by not logging gas function call. Logging `gas` was a intention choice because i thought it would be helpful to stat its use but betanet deployment has shown it slowdown the node significantly.

Test Plan
------------
same as #5018, this time the output doesn't have `gas`:
```
Oct 25 16:55:57.767  INFO stats: Server listening at ed25519:5V9BRNjjF7FvHBMH4QRWyeQHnNVhGj7rxnHVTbVmt2Wj@0.0.0.0:24567
Oct 25 16:56:01.702 DEBUG input: host-function: enter
Oct 25 16:56:01.702 DEBUG input: host-function: close time.busy=225µs time.idle=29.7µs
Oct 25 16:56:01.702 DEBUG register_len: host-function: enter
Oct 25 16:56:01.702 DEBUG register_len: host-function: close time.busy=118µs time.idle=20.2µs
Oct 25 16:56:01.702 DEBUG read_register: host-function: enter
Oct 25 16:56:01.702 DEBUG read_register: host-function: close time.busy=117µs time.idle=28.8µs
Oct 25 16:56:01.703 DEBUG storage_read: host-function: enter
Oct 25 16:56:01.703 DEBUG storage_read: host-function: close time.busy=330µs time.idle=12.9µs
Oct 25 16:56:01.703 DEBUG register_len: host-function: enter
Oct 25 16:56:01.703 DEBUG register_len: host-function: close time.busy=60.5µs time.idle=11.1µs
Oct 25 16:56:01.703 DEBUG read_register: host-function: enter
Oct 25 16:56:01.703 DEBUG read_register: host-function: close time.busy=72.8µs time.idle=11.1µs
Oct 25 16:56:01.704 DEBUG signer_account_id: host-function: enter
Oct 25 16:56:01.704 DEBUG signer_account_id: host-function: close time.busy=66.7µs time.idle=11.5µs
Oct 25 16:56:01.704 DEBUG register_len: host-function: enter
Oct 25 16:56:01.704 DEBUG register_len: host-function: close time.busy=61.3µs time.idle=10.9µs
Oct 25 16:56:01.704 DEBUG read_register: host-function: enter
Oct 25 16:56:01.704 DEBUG read_register: host-function: close time.busy=72.0µs time.idle=13.8µs
Oct 25 16:56:01.704 DEBUG log_utf8: host-function: enter
Oct 25 16:56:01.704 DEBUG log_utf8: host-function: close time.busy=74.5µs time.idle=11.5µs
Oct 25 16:56:01.704 DEBUG storage_write: host-function: enter
Oct 25 16:56:01.704 DEBUG storage_write: host-function: close time.busy=190µs time.idle=12.2µs
Oct 25 16:56:07.820  INFO stats: #    3339 332KuomX4nwiREdZwRLA63uekeWLnfszv6UtGctn8Tgf V/1  0/0/40 peers ⬇ 0 B/s ⬆ 0 B/s 1.60 bps 543.20 Ggas/s CPU: 0%, Mem: 0 B  
```

Also @mina86 is there a way to test performance as betanet node before merge it to master?

### CODE REVIEW

**Comment by ailisp** [comment]
@mina86 @matklad finished test by run a testnet node. cpu and memory usage is not higher than master

File: runtime/near-vm-runner/src/imports.rs [hunk 1]
```diff
@@ -41,11 +57,18 @@ macro_rules! wrapped_imports {
...
                 $(
                     #[allow(unused_parens)]
                     $(#[cfg(feature = $feature_name)])*
                     pub fn $func( ctx: &mut Ctx, $( $arg_name: $arg_type ),* ) -> VMResult<($( $returns ),*)> {
+                        const IS_GAS: bool = str_eq(stringify!($func), "gas");
+                        if IS_GAS {
+                            None
+                        } else {
+                            Some(tracing::debug_span!(target: "host-function", stringify!($func)).entered())
+                        };
```
Comment (matklad) [inline runtime/near-vm-runner/src/imports.rs:71]: Wait, this doesn't do what we want it to. This `;` immediately drops the span, *before* the host function is even invoked. We do need to bind it to a temporary variable to make sure it is dropped after `$func` returns: 

```rust
                        let _span = if IS_GAS {
                            None
                        } else {
                            Some(tracing::debug_span!(target: "host-function", stringify!($func)).entered())
                        };
```

**Review by matklad — CHANGES_REQUESTED** [review]

File: runtime/near-vm-runner/src/imports.rs (hunk 1 above)
Comment (ailisp) [reply runtime/near-vm-runner/src/imports.rs:71]: ah yes. Sorry

Comment (ailisp) [reply runtime/near-vm-runner/src/imports.rs:71]: Test again on testnet, with correctly set _span the time of certain host functions looks realistic!

**Review by mina86 — APPROVED** [review]
LGTM

**Review by matklad — APPROVED** [review]

**Pull request merged**